music21/analysis/reduceChordsOld.py

```diff
@@ -79,15 +79,14 @@ def reduceMeasureToNChords(self,
         4.0
         '''
         from music21 import note
-        if measureObj.isFlat is False:
+        if not measureObj.isFlat:
```

Comment:
Please revert -- this is probably fine, but to be sure would need to brush up on whether measureObj is ever None and if so what is appropriate behavior then.

Reply:
It is never `None`, you can check by searching the codebase for any assignment to `isFlat`.
Will revert anyways

Reply:
Oh, for sure, I know I can check, I'm just saying I'd prefer not to in this PR. :-)